data/json/items/armor/helmets.json

```diff
@@ -306,6 +306,36 @@
     ],
     "melee_damage": { "bash": 2 }
   },
+  {
+    "id": "helmet_soviet_tanker",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "soviet tanker helmet" },
+    "description": "A soviet era tanker helmet.  It's distinctive foam pads are meant to protect a vehicle driver from bumps against the cramped interior.",
```

Comment:
```suggestion
    "description": "A Soviet era tanker helmet.  It's distinctive foam pads are meant to protect a tank's driver from bumps against the cramped interior.",
```

Reply:
Sure, sounds good to me. Is the description change mandatory or just a suggestion though?

Reply:
Not mandatory and I could be wrong.  Considering Soviet is short for Soviet Union I thought it should probably be capitalized too.

Reply:
I'll keep Soviet capitalized, I think it's correct but the replacement of vehicle by tank is a good suggestion.